app/services/ssoe/get_ssoe_traits_by_cspid_message.rb

```diff
@@ -0,0 +1,104 @@
+# frozen_string_literal: true
+
+module SSOe
+  class GetSSOeTraitsByCspidMessage
+    attr_reader :credential_method,
+                :credential_id,
+                :first_name,
+                :last_name,
+                :birth_date,
+                :ssn,
+                :email,
+                :phone,
+                :street1,
+                :city,
+                :state,
+                :zipcode
+
+    def initialize(**params)
```

Comment:
I think keyword arguments are important here but I get why you did this since there are so many. If we do args like this we should add some sort of validation to make sure all the required fields are passed. But imo it's usually cleaner to just pass keyword args. Another option would be to use ActiveModel and ActiveModel::Attributes